src/App.js

```diff
@@ -0,0 +1,231 @@
+export class App {
+
+    todoList = [];
+    todoFilter = "all";
+    todoComplete = [];
+
+    constructor({ }) {
+        this.$newTodo = document.querySelector('#new-todo-title');
+        this.$todoList = document.querySelector('#todo-list');
+        this.$todoCount = document.querySelector('span.todo-count strong');
+        this.$seeAllButton = document.querySelector('a.all');
+        this.$seeActiveButton = document.querySelector('a.active');
+        this.$seeCompletedButton = document.querySelector('a.completed');
+
+        this.$newTodo.addEventListener('keyup', event => {
+            if (event.key === 'Enter') {
+                this.addTodo(this.$newTodo.value);
+            }
+        })
```

Comment:
constructor에서 굉장히 많은 일들이 일어나고 있어서 이 부분이 가독성을 해치는거 같습니다!

 eventListener에 함수를 등록할 때 인라인 함수로 등록하지 않고 constructor 바깥의 별도의 함수로 분리하고 등록하게 되면 가독성이 훨씬 좋아질거 같습니다!

```suggestion
        this.$newTodo.addEventListener('keyup', newTodoTitle)
        
        newTodoTitle = (event) => {
            if (event.key === 'Enter') {
                this.addTodo(this.$newTodo.value);
            }
        }
```

Reply:
좋은 생각이네요. 반영해보겠습니다!